src/olympia/devhub/templates/devhub/addons/listing/items.html

```diff
@@ -14,7 +14,7 @@
           {{ _("This add-on doesn't have any versions.")}}
         </p>
       {% else %}
-        {% if addon.status != amo.STATUS_NULL %}
+        {% if addon.status != amo.STATUS_NULL and addon.has_listed_versions() %}
```

Comment:
I don't understand this change. If an addon's status isn't NULL then it's pretty much got to have listed_versions, so isn't `and addon.has_listed_versions()` redundant?  Is it supposed to be `or`?